yarn.lock

```diff
@@ -97,7 +97,6 @@
 "@types/anymatch@*":
```

Comment:
I am not sure if you intentionally wanted to change yarn.lock. If not maybe undo changes to it so it is not part of the pull request?

Reply:
This was due to changing `package.json` (I changed the hard version pin on `redux-data-service` from `0.0.16` to `0.0.32`, the most recent release). When we update `package.json`, we do want to include the associated changes in the `yarn.lock` file.